Userland/Libraries/LibWeb/HTML/Parser/HTMLToken.h

```diff
@@ -40,9 +40,9 @@ class HTMLToken {
 
     struct Attribute {
         String prefix;
-        String local_name;
+        String local_name { "" };
         String namespace_;
-        String value;
+        String value { "" };
```

Comment:
This doesn't seem correct. How do we distinguish between `<tag attr="">` and `<tag attr>` now?

Reply:
I think `<tag attr>` is supposed to be equivalent to `<tag attr="">`, since an attribute with a missing value implicitly has an empty string as its value (https://html.spec.whatwg.org/multipage/syntax.html#attributes-2). If these are not initialized with `""`, the HTML element seems to have no way of distinguishing between an attribute with a missing value vs. having no such attribute, since in both cases the string is null.

Reply:
Ah gotcha, that makes sense :)